apps/frontpage/next.config.js

```diff
@@ -53,6 +53,11 @@ module.exports = {
         destination: '/docs/configure/telemetry',
         permanent: true,
       },
+      {
+        source: '/status',
+        destination: 'https://github.com/storybookjs/storybook/issues/23279',
+        permanent: true,
```

Comment:
I really think this should be `false`. We know we don't want to redirect here forever.